tests/test_modeling_tf_common.py

```diff
@@ -1881,16 +1881,22 @@ class DummyModel:
             def __init__(self):
                 config_kwargs = {"output_attentions": False, "output_hidden_states": False, "return_dict": False}
                 self.config = PretrainedConfig(**config_kwargs)
+                self.main_input_name = "input_ids"
 
             @unpack_inputs
             def call(
                 self, input_ids=None, past=None, output_attentions=None, output_hidden_states=None, return_dict=None
             ):
                 return input_ids, past, output_attentions, output_hidden_states, return_dict
 
+            @unpack_inputs
+            def foo(self, pixel_values, output_attentions=None, output_hidden_states=None, return_dict=None):
```

Comment:
Is the `foo` thing something you did while testing, or are we going to keep it?

I'm fine either way, I just know I have a tendency to accidentally leave `breakpoint()` in code and then push it, so I wanted to make sure this wasn't one of those cases!

Reply:
It's intentional -- to test on a function whose first arg is not `self.main_input_name` (which was the issue that led to this PR :D )